src/snyk/common/languageServer/middleware.ts

```diff
@@ -40,7 +42,16 @@ export class LanguageClientMiddleware implements Middleware {
       }
 
       const serverSettings = await LanguageServerSettings.fromConfiguration(this.configuration);
-      return [serverSettings];
+
+      return [
+        {
+          ...serverSettings,
+          integrationName: CLI_INTEGRATION_NAME,
+          integrationVersion: await Configuration.getVersion(),
+          deviceId: this.user.anonymousId,
+          automaticAuthentication: 'false',
```

Comment:
question: why not integrate these values into server settings? Right now, if I understand the change correctly, these settings remain apart from the "normal" server settings. Is there a need for that? If necessary, I'd rather update LS to export these settings, as after initialization it should know them.

Reply:
Oh, I see!! I think I misunderstood the suggested change written in the ticket :/

Updating it 💪